packages/frontend-2/components/projects/AddDialog.vue

```diff
@@ -30,6 +30,7 @@
         <ProjectVisibilitySelect
           v-model="visibility"
           class="sm:max-w-none w-full sm:w-80"
+          mount-menu-on-body
```

Comment:
Just tested it in 1 dialog for now, @andrewwallacespeckle can use it as needed elsewhere